src/components/utils.tsx

```diff
@@ -204,5 +204,5 @@ export const VerticalDash = () => {
     const { primaryColor } = useContext(DatepickerContext);
     const bgColor = BG_COLOR["500"][primaryColor as keyof (typeof BG_COLOR)["500"]];
 
-    return <div className={`bg-blue-500 h-7 w-1 rounded-full hidden md:block ${bgColor}`} />;
+    return <div className={`h-7 w-1 rounded-full hidden md:block ${bgColor??'bg-blue-500'}`} />;
```

Comment:
There is indeed a problem. But I think the solution would be:

```
`h-7 w-1 rounded-full hidden md:block ${bgColor || 'bg-blue-500'}`
```

Reply:
I think I'll solve it myself as I don't know how long it will take you to solve it.